autogluon/task/object_detection/nets.py

```diff
@@ -10,7 +10,7 @@ def _get_network(meta_arch, name, transfer_classes, ctx=mx.cpu()):
             net = gcv.model_zoo.get_model(net_name,
                                           classes=transfer_classes,
                                           pretrained_base=False,
-                                          transfer='coco')
+                                          transfer=None)
```

Comment:
why change to `None` this time?

Reply:
It doesn't matter here since we will load our trained model later and all weights will be updated. I just change it to None to make it consistent with faster-rcnn part
https://github.com/chongruo/autogluon/blob/d9cb58c2762fbcd496d3fec7ade57416fe2707bf/autogluon/task/object_detection/nets.py#L29